proto/Cargo.toml

```
@@ -36,5 +36,6 @@ futures.workspace = true
 
 tokio-openssl.workspace = true
 openssl.workspace = true
+sspi = { workspace = true, opional = true }
```

That’s because there is a typo:
```suggestion
sspi = { workspace = true, optional = true }
```